src/components/AccessibilityGuide/AriaLandmarks.mdx

```diff
@@ -0,0 +1,125 @@
+
+### form
+
+The `form` landmark identifies a region that contains a collection of elements that combine to create a form: usually `button`, `input`,
+`select`, and `textarea`.
+
+<Banner
+  content={
+    <>
+      <strong>IMPORTANT:</strong> Use the search landmark instead of the form
+      landmark when the form is used for search functionality.
+    </>
+  }
+  type="warning"
+/>
+
+Interfaces can have multiple `form` regions, and each should be identified with a unique label.
+
+Elements under [Form Elements](../?path=/docs/components-form-elements) should be wrapped by the `form` landmark.
+
+### main
+
+The `main` landmark is meant to designate the primary content of a page. In most browsers, it's natively conveyed by use of the `<main>` HTML element.
+
+There should only be a single `main` landmark per interface. Since `main` represents the primary content of a page, it is the most important destination for [skip links](../?path=/docs/accessibility-guide-skip-navigation--docs), bypassing recurring elements like the header or navigation.
+
+The `Template` [component](../?path=/docs/components-page-layout-template--docs) renders the `main` landmark (and the `header` and `footer`).
+
+### navigation
+
+The `navigation` landmark identifies groups (e.g. lists) of links that are intended to be used for website or page content navigation.
+There can be multiple `navigation`s on a page, but if a `navigation` landmark has the same set of links as another `navigation` landmark on the page (think mobile versus desktop), use the same HTML `label` attribute for both.
+
+Elements under [Navigation](../?path=/docs/components-navigation) should be wrapped by the `navigation` landmark.
+
+### region
+
+The `region` landmark is sort of a catch-all: a perceivable section containing content that is sufficiently important that users may want to be able to navigate to the section directly, and to have it listed in a summary of the page.
+
+Since `region` is for content that doesn't fit under other landmarks, it has no default corresponding HTML element. A `<section>` element can become a `region` landmark if it is provided an accessible name (for instance, by giving it an `aria-label` or `aria-labelledby` attribute).
+
+`region` should always be applied by the consuming app, since it is specific to the content being presented.
+
+Note: There's not always a clear line between `complementary` content and content that deserves its own `region`, but a good rule of a thumb is that `complementary` content should support or modify the main content of the page, while `region` should represent its own separate information.
+
+### search
+
+The `search` landmark contains the page's search and filter functionality, which will likely include form controls (such as an `input type="search"`).
+
+Inspect below to see `<search>` in the DOM– notice that it wraps the `Heading`, the `TextInput`, the `Button`, and the `Checkbox`, not just the searchbar:
```

Comment:
Replace the dash with an em dash and remove the trailing space.

If we are going to ask users to look for the DS components, let's tell them to click on the `Show code` button in the lower right corner. Conversely, if we are going to ask them to inspect the DOM, then let's ask them to look for the native HTML elements.